tests/resources/sample_remote_repo_expected_history_churn.json

```diff
@@ -0,0 +1,93 @@
+[
+  {
+    "timestamp": "2022-08-26 04:55:09",
```

Comment:
Question here: on my local all timestamp from `gto history` are shown shifted 2 hours forward (e.g. this one is at 6AM).
On GitHub actions I believe they use UTC. So, I changed the timestamp in this json file accordingly, and now the CI is not broken anymore. BUT on my local machine this test is failing, and so on most people's computer, I fear.

I think this should be fixed, and the timestamp should specify the time zone, or refer always to UTC... any idea?

Reply:
Hmm, let me check on my Mac.

Reply:
Alright, reproduced this. Few options I see so far:
1. Exclude the `timestamp` field from check. Could be done with `for` loop + `tests.utils.check_obj(..., skip_keys=["timestamp"])` I believe. This is what I did before e.g. for `test_registry`. This is also handy since you don't need to change `resources` and fix tests after the test repo is re-generated.
2. Fix datetimes in GTO to be TZ-aware, and then before you `dump` them, make them be in UTC. Will require something like https://github.com/iterative/gto/blob/main/gto/utils.py#L24 I suppose, which will discover datetimes inside a nested objects and serialize them. (Fixing them to be tz-aware is a needed thing on it's own, but only for API consumers like Studio BE, so this is not a direct benefit for this item so far.)

Reply:
3. Change the timezone for the tests. Looking if this possible. Maybe somehow manupulate shell ENV vars for the `pytest` process, or mock the `datetime` module in Python, dunno yet :)

Reply:
Looks like I found the answer thanks to this [SO answer](https://stackoverflow.com/a/48051938/4890419). Pushed the code, you can check if that works for you. Works for me, local tests pass as expected.

Reply:
I guess I'll need to decide on which approach is better and update the tests to force it.

Reply:
nice, thanks!

what? I thought that frozen means frozen! :D